api/src/services/data-context.js

```diff
@@ -0,0 +1,7 @@
+const { getLocalDataContext } = require('@medic/cht-datasource');
+const db = require('../db');
+const config = require('../config');
+
+const dataContext = getLocalDataContext(config, db);
+
+Object.assign(module.exports, dataContext);
```

Comment:
I like the brevity of this pattern but I would prefer if it was [greppable](https://mrcoles.com/is-it-greppable/). It's a bit more tedious to re-export each property but explicit > implicit here

Reply:
This is good advice. We can be explicit here with what we are assigning and so we should be.

Reply:
Actually, looking at this again, I guess we can simplify this code here by just directly exporting the dataContext object, itself, and not trying to re-bind anything to any new object....